libs/barista-components/chart/src/chart-options.ts

```diff
@@ -56,6 +56,11 @@ export const DT_CHART_DEFAULT_OPTIONS: DtChartOptions = {
   },
   plotOptions: {
     series: {
+      states: {
+        inactive: {
+          opacity: 1,
```

Comment:
NIT: It would be really awesome to add a comment here what this option does, otherwise I'll probably forget until early next week. 
```suggestion
          // This handles enabling / disabling the inactive state
          // Inactive state happens for example when a series has data missing (null as value for datapoints)
          // by default the series in area charts gets hidden up from v7. Setting this to 1 disables this behavior 
          opacity: 1,
```

Reply:
Done!